nautobot_netbox_importer/diffsync/models/virtualization.py

```diff
@@ -24,7 +24,6 @@ def setup(adapter: SourceAdapter) -> None:
     adapter.configure_model(
         "virtualization.vminterface",
         fields={
-            "status": "status",
```

Comment:
I understand that status is added to all the models with status, by default?

Reply:
Correct, as described above.